app/templates/posts.hbs

```diff
@@ -24,8 +24,8 @@
                                         {{#if post.page}}
                                             <span class="page">Page</span>
                                         {{else}}
-                                            <time datetime="{{post.publishedAtOffset}}" class="date published">
-                                                Published {{gh-format-timeago post.publishedAt}}
+                                            <time datetime="{{post.publishedAt}}" class="date published">
+                                                Published {{gh-format-timeago post.publishedAt}} ago
```

Comment:
Found the bug: I used `fromNow(moment.utc())` although `fromNow()` only excepts a boolean value. Changed it to `from(moment.utc())` as described [here](http://momentjs.com/docs/#/displaying/from/) in the moment docs and it works fine now.